docs/_articles/en/v3.x/release-information.md

```diff
@@ -5,6 +5,19 @@ lang: en
 
 ### To update Code Analyzer, follow these [instructions](./en/v3.x/getting-started/install/#upgrade-plug-in)
 
+## [v3.5.1](https://github.com/forcedotcom/sfdx-scanner/tree/v3.5.1) (09-26-2022)
+
+[Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v3.5.0...v3.5.1)
+
+### Release Summary
+
+* NEW: We made some updates to the RetireJS vulnerability database.
+* CHANGE: SFGE now emits telemetry data when unsupported scenarios are encountered.
```

Comment:
@jfeingold35 I thought we decided that telemetry wasn't going to have release notes, that it was "under-the-hood" and not public release noteable?

If we are going to release note telemetry, @johnbelosf and Woodson don't want us using SFGE or SFCA anywhere in public doc. Please use Salesforce Graph Engine, or Graph Engine for short, or Salesforce Code Analyzer, or Code Analyzer for short. Cool? Thanks!

So here:

CHANGE: Salesforce Graph Engine now emits telemetry data when unsupported scenarios are encountered.